src/main/java/com/synopsys/integration/alert/channel/azure/boards/descriptor/AzureBoardsGlobalUIConfig.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.channel.azure.boards.descriptor;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
+import com.synopsys.integration.alert.common.descriptor.config.field.PasswordConfigField;
+import com.synopsys.integration.alert.common.descriptor.config.field.TextInputConfigField;
+import com.synopsys.integration.alert.common.descriptor.config.field.URLInputConfigField;
+import com.synopsys.integration.alert.common.descriptor.config.field.endpoint.EndpointButtonField;
+import com.synopsys.integration.alert.common.descriptor.config.field.validators.EncryptionValidator;
+import com.synopsys.integration.alert.common.descriptor.config.ui.UIConfig;
+
+@Component
+public class AzureBoardsGlobalUIConfig extends UIConfig {
+    public static final String LABEL_AZURE_BOARDS_URL = "Url";
+    public static final String LABEL_ORGANIZATION_NAME = "Organization Name";
+    public static final String LABEL_CONSUMER_KEY = "Consumer Key";
+    public static final String LABEL_PRIVATE_KEY = "Private Key";
+    public static final String LABEL_OAUTH = "Initialize OAuth";
+    public static final String LABEL_ACCESS_TOKEN = "Access Token";
+
+    public static final String DESCRIPTION_AZURE_BOARDS_URL = "FILL OUT THIS DESCRIPTION";
```

Comment:
At least a FIXME should be added here just so we are aware this isn't finished yet.